iina/MainWindowController.swift

```diff
@@ -2584,7 +2570,10 @@ class MainWindowController: PlayerWindowController {
       if let screenFrame = window.screen?.frame {
         rect = rect.constrain(in: screenFrame)
       }
-      if player.disableWindowAnimation || Preference.bool(for: .disableAnimations) {
+      if !player.mainWindow.window!.isVisible && !player.isInMiniPlayer {
+        window.setFrame(rect, display: false, animate: false)
+        showWindow(nil)
```

Comment:
Why put it here in `adjustFrameByVideoSize()`? This can be called whenever the video's size is changed.

Consider the following case: I open an IINA window and hide it using Cmd+H. Then it automatically continues to the next video (or some plugin/mpv script changed the video/window size), and this line will show the window unexpectedly.